### ISSUE TITLE
Cannot set color and opacity using healthicons-react-native

### ISSUE THREAD

**Issue by daankennes** [stnrd/healthicons#21]
Hi! Thank you for your work.

I just installed healthicons-react-native 2.0.2 in my project. Setting height and width works, but other props do not seem to have an effect. For example, I tried:

```
import {PrescriptionDocument} from 'healthicons-react-native';
...
<PrescriptionDocument
                    color="#fff"
                    fill="'#fff'"
                    opacity={0.85}
                    height={24}
                    width={24}
                />
```

but except height/width none of these props work.

I am using Expo.

Would be great to hear some feedback!

Thank you,
Daan

**Comment by stnrd** [comment]
I found a bug in my generation code which should remove the fill element from the source svg. This wasn't happening for react native therefore always overwritten with black. 

I have solved it in this PR #22 and the changes are released in 2.0.3. 

I have adjusted the Expo example application with color and opacity.(examples/react-native)

**Comment by daankennes** [comment]
Wow that was quick! Works like a charm. Thank you!
